docs/examples/frameworks/jax/flax-basic_example.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>In this section "Getting started" link redirected me to the raw jupyter notebook and "pipeline documentation" to the raw rst file. Could you double-check these?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4978/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yes, the review tool can't handle these links. I checked them in the docs build.